File: src/dotnet/ArdsTools/Ards.NetCore.Entity/Models/RegionApplicability.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace Ards.NetCore.Entity.Models
{
    [Table("region_applicability", Schema = "adif")]
    public partial class RegionApplicability
    {
        [Column("id")]
        public int Id { get; set; }
        [Column("region_id")]
        public int RegionId { get; set; }
        [Column("weblink_id")]
        public int WeblinkId { get; set; }
    }
}

File: src/java/ArdsToolAdifModels/src/com/ards/jpa/model/RegionApplicability.java
package com.ards.jpa.model;

import java.io.Serializable;
import javax.persistence.*;


/**
 * The persistent class for the region_applicability database table.
 * 
 */
@Entity
@Table(name="region_applicability")
@NamedQuery(name="RegionApplicability.findAll", query="SELECT r FROM RegionApplicability r")
public class RegionApplicability implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	@GeneratedValue(strategy=GenerationType.AUTO)
	@Column(unique=true, nullable=false)
	private Integer id;

	@Column(name="region_id", nullable=false)
	private Integer regionId;

	@Column(name="weblink_id", nullable=false)
	private Integer weblinkId;

	public RegionApplicability() {
	}

	public Integer getId() {
		return this.id;
	}

	public void setId(Integer id) {
		this.id = id;
	}

	public Integer getRegionId() {
		return this.regionId;
	}

	public void setRegionId(Integer regionId) {
		this.regionId = regionId;
	}

	public Integer getWeblinkId() {
		return this.weblinkId;
	}

	public void setWeblinkId(Integer weblinkId) {
		this.weblinkId = weblinkId;
	}

}